apps/studio/data/database-policies/keys.ts

```diff
@@ -1,4 +1,4 @@
 export const databasePoliciesKeys = {
   list: (projectRef: string | undefined, schema?: string | undefined) =>
-    ['projects', projectRef, 'database-policies', schema] as const,
+    ['projects', projectRef, 'database-policies', schema].filter(Boolean),
```

Comment:
just to check why we're doing this?

Reply:
When we want to invalidate `[projects, <ref>]` (and all its children like `[projects, <ref>, storage]`) we're using `invalidateQueries(databasePoliciesKeys.list(projectRef))`. The issue is in `databasePoliciesKeys.list(projectRef)` called with `schema: undefined` (because it's optional) which generates `[projects, <ref>, undefined]` which won't invalidate the storage schema (since it's a sibling key, not a parent key in the RQ cache). 

This is a really obscure bug but you can replicate it on production: 
1. Add a RLS policy to `storage` schema
2. Refresh
3. Go to `storage/policies`, let it load the policies
4. Go to `auth/policies`, select the storage schema
5. Delete the added policy
6. Go to `storage/policies` and you can still see the policy because the cache hasn't been invalidated

Reply:
ahhhhh okay gotcha on this yup this makes sense - we probably might need to do the .filter(Boolean) in several other places then, this sounds like a viable long term solution